.circleci/config.yml

```diff
@@ -0,0 +1,211 @@
+alias:
+  default: &default
+    working_directory: /go/src/github.com/yahoojapan/garm
+    docker:
+      - image: circleci/golang:1.11.2
+        environment:
+          GOPATH: "/go"
+          GO111MODULE: "on"
+          REPO_NAME: "yahoojapan"
+          IMAGE_NAME: "garm"
+          GITHUB_API: "https://api.github.com/"
+          DOCKER_USER: "yahoojapancorp"
+  setup_remote_docker: &setup_remote_docker
+    version: 18.05.0-ce
+    docker_layer_caching: true
+
+version: 2
+jobs:
+  test:
+    <<: *default
+    steps:
+      - checkout
+      - restore_cache:
+          key: gosum-{{ .Branch }}-{{ checksum "go.sum" }}
+      - run:
+          name: preparation
+          command: |
+            go mod vendor
+      - run:
+          name: run tests
+          command: |
+            go test -v -race -covermode=atomic -coverprofile=coverage.out ./...
+            go tool cover -html=coverage.out -o coverage.html
+      - store_artifacts:
+          path: ./coverage.html
+      - save_cache:
+          key: gosum-{{ .Branch }}-{{ checksum "go.sum" }}
+          paths:
+            - ./vendor
+  build:
+    <<: *default
+    steps:
+      - setup_remote_docker: *setup_remote_docker
+      - checkout
+      - run:
+          name: docker image build
```

Comment:
Please enable DockerKit for parallel build

Reply:
Thanks.
I updated to Docker 18.06, and added `DOCKER_BUILDKIT=1`. However, unfortunately it is not supported yet in CircleCI.
https://circleci.com/gh/yahoojapan/garm/12